core/src/services/oss/backend.rs

```diff
@@ -253,6 +257,28 @@ impl OssBuilder {
         };
         Ok((endpoint, host))
     }
+
+    /// Set server_side_encryption for this backend.
+    ///
+    /// Available values: `AES256`, `KMS`.
```

Comment:
Can you provide more details about the meaning of this different value? For example, link to https://www.alibabacloud.com/help/en/object-storage-service/latest/server-side-encryption-5 and explain what's `SSE-KMS` and `SSE-OSS`.